protobuf-provider/src/main/java/io/confluent/kafka/schemaregistry/protobuf/ProtobufSchemaUtils.java

```diff
@@ -107,6 +110,80 @@ public static byte[] toJson(Message message) throws IOException {
     return jsonString.getBytes(StandardCharsets.UTF_8);
   }
 
+  public static ProtoFileElement jsonToFile(JsonNode node) throws JsonProcessingException {
+    ObjectMapper mapper = new ObjectMapper();
```

Comment:
Creating an ObjectMapper is expensive, so this should be moved to a static variable.